pkg/matcher/matcher_test.go

```diff
@@ -48,16 +48,16 @@ func TestTextMatcher_ValidateFails(t *testing.T) {
 
 func TestEqualMatcher_Validate(t *testing.T) {
 	m := EqualMatcher{}
-	got := m.Match(1, 1)
+	got := m.Match(2, 2)
 	assert.True(t, got.Success)
 }
 
 func TestEqualMatcher_ValidateFails(t *testing.T) {
 	m := EqualMatcher{}
-	got := m.Match(1, 0)
+	got := m.Match(2, 3)
 	assert.False(t, got.Success)
-	assert.Contains(t, got.Diff, "+0")
-	assert.Contains(t, got.Diff, "-1")
+	assert.Contains(t, got.Diff, "+3")
+	assert.Contains(t, got.Diff, "-2")
```

Comment:
I noticed that there was always a "-1" in the diff output because it tells the user how many lines changed, so this makes the test a tiny bit less brittle.